tests/functional/test_s3.py

```diff
@@ -467,16 +459,16 @@ def test_correct_url_used_for_s3():
 
     # The default behavior for sigv2. DNS compatible buckets
```

Comment:
We should probably update the comment to be about just DNS compatible bucket names and not mention sigv2. Also for the test cases, it may make sense to just remove the `signature_version` from the cases because we already have cases where we explicitly use `s3v4` in the next set of cases below this set of cases.

Reply:
Okay, I will rebase and incorporate feedback.  Thanks.